.github/workflows/auto-update.yml

```
@@ -0,0 +1,150 @@
+            echo "has_changes=true" >> $GITHUB_OUTPUT
+            echo "Changed files:"
+            git diff --name-only
+          fi
+
+  test:
+    if: needs.check-and-update.outputs.has_changes == 'true'
+    uses: ./.github/workflows/test-sdk.yml
+    needs: check-and-update
+    with:
+      node-versions: '[18, 19, 20, 21, 22]'
+      upload-coverage: 'false'
```

actionlint (1.7.7)

input "upload-coverage" is typed as string by reusable workflow "./.github/workflows/test-sdk.yml". bool value cannot be assigned

(expression)